acceptance/bundle/resource_deps/job_tasks/script

```diff
@@ -1,4 +1,8 @@
+if [ "$DATABRICKS_BUNDLE_ENGINE" == "direct" ]; then
+    update_file.py databricks.yml 'task[0].python_wheel_task[0].' 'tasks[0].python_wheel_task.'
```

Comment:
Please include a comment why this is necessary.

IIRC, it was about TF resolving paths differently for nested blocks.